src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -8,6 +8,16 @@
 
 namespace System
 {
+   public partial class AppDomain
```

Comment:
I'm good going with that as the plan until we find a reason it doesn't work (if we find one).